redfish-core/lib/log_services.hpp

```diff
@@ -1197,6 +1197,14 @@ inline void createDumpTaskCallback(
             "member='PropertiesChanged',path='" +
                 createdObjPath.str + "'");
 
+        // Take the task state to "Running" for all dumps except
+        // Resource dumps as there is no validation on the user input
+        // for dump creation, meaning only in resource dump creation,
+        // validation will be done on the user input.
+        if ((createdObjPath.str).find("/system/entry/B") == std::string::npos)
```

Comment:
B is a random string? How do we know it is a resource dump ?

Reply:
Dhruv has this design doc that has these defined. Dump ids starting with:
A - System dump
B - Resource dump
0 - Hardware dump
2 - Hostboot dump
3 - SBE dump
4 - OCMB SBE dump

So I have made this change as per (Dhruv's) backend dump design.

Reply:
Perhaps, can we define those prefixes as macros like?
```
constexpr std::string  prefixResourceDump = "/xyz/openbmc_project/dump/system/entry/B";
if ((createdObjPath.str).starts_with(prefixResourceDump) == std::string::npos)
```

Reply:
I saw in slack that you will add the design document link here when Dhruv is back tomorrow. Thank you!

Reply:
This isn't great from an API standpoint.. Magic numbers/letters have a hidden meaning. :/  
```
A - System dump
B - Resource dump
0 - Hardware dump
2 - Hostboot dump
3 - SBE dump
4 - OCMB SBE dump
```